ui/src/app/components/help-sidebar.tsx

```diff
@@ -528,14 +574,20 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile(), wi
             <div style={contentStyle('annotations')}>
               {participant && <SidebarContent />}
             </div>
-            <div style={styles.footer}>
+            <div style={contentStyle('criteria')}>
+              {!!attributesSelection
+                ? <AttributesPageV2 close={() => attributesSelectionStore.next(undefined)} node={attributesSelection}/>
+                : <SelectionList back={() => {}} selections={[]}/>
+              }
+            </div>
```

Comment:
This probably has some overlap with #3822. I needed it here to include the transition between the selection list and attributes form